### DISCUSSION TITLE
Descrições de pontos do mapa que estão faltando no Mapa Interativo de Arton

### DISCUSSION THREAD

**Discussion by YuriAlessandro** [YuriAlessandro/gerador-ficha-tormenta20#367] [Mapa Interativo de Arton]
A maior parte das descrições foram retiradas do livro Atlas de Arton, então são a fonte mais recente e oficial possível.

Entretanto, vários locais não possuem descrição no livro. Se você deseja adicionar a descrição de um local, você pode responder a este tópico, no seguinte formato:

<hr>

**Nome do local:**
**Reino (se tiver):**
**Descrição sugerida:** _Insira a descrição bem formatada, resumida e que mostre os pontos mais importantes daquele local._
**Fonte:** _Tente utilizar a fonte mais recente possível, e OFICIAL da Jambô. Textos não oficiais não serão considerados. Todas as fontes serão checadas_

<hr>

**Comment by Axgaa** [comment]
Lar de Marah, uma comunidade que conta com templos, dormitórios e hortas, seus habitantes são soldados aposentados, ex-criminosos, pessoas que querem paz na vida. O Lar as vezes é atacado por servos de Keenn e Sszzaas, ou pessoas contratadas por eles.

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by Sapo-de-Terno** [comment]
**Nome do Local:** Ruínas de Kharadantur
**Reino:** Doherimm
**Descrição Sugerida:** "Próxima a uma floresta de cogumelos, Kharadantur foi palco do primeiro ataque Trollkyrka a Doherimm, em 1203. A investida começou com o envio de hordas de mycotann lobotomizados magicamente, preparando - com um ataque suicida - o terreno para a invasão direta e massiva das tropas finntroll que se seguiu. Essa brusca manobra devastou a cidade e marcou oficialmente o início da permaguerra."
**Fonte:** Penúltimo parágrafo do trecho "Amigos e Inimigos" do capítulo de Doherimm, página 409 do livro (411 do pdf)

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by Gramogoblin** [comment]
Nome do Local: Cabana de Ian
Reino: Montanhas Uivantes
Descrição Sugerida:
Dizem que lan, o Ermitão, era um próspero pescador de Trebuck - até que testemunhou, além do Rio dos Deuses, o surgimento de uma área de Tormenta que até hoje ameaça o reino. A visão perturbou tanto sua mente que ele teria ficado louco, e decidiu viver aqui pelo resto de seus dias. Isso, pelo menos, é o que dizem. Mas alguns aventureiros garantem que lan tem poderes mágicos adivinhatórios. Ele seria capaz de revelar a localização de pessoas, criaturas ou itens perdidos em qualquer pontos das Uivantes. Mas conseguir esse favor é um problema, pois lan gosta de ficar sozinho e costuma provocar desmoronamentos sobre qualquer pessoa que tente se aproximar de sua cabana."

Fontes: O Reinado 3D&T - Parte 2, página 108. / O Reinado - Tormenta D20, página 142. (Retirado de um comentário no reddit: https://www.reddit.com/r/Tormenta_20/comments/18g0j74/cabana_de_ian/)

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by Gramogoblin** [comment]
Nome do Local: Palácio de Laponya
Reino: Montanhas Uivantes
Descrição sugerida:
"Igualmente intrigante é o Palácio de Laponya, belíssima qareen de pele azulada e resplandecente com cristais de gelo, o cabelo branco como pelagem de raposa das neves. Veste luxuosas peles e joias de ouro branco, uma névoa leitosa acompanhando seus movimentos quando surge nas sacadas marmóreas. Até onde se sabe, ela vive ali sozinha — mas em certas ocasiões os ventos que sopram da estrutura parecem trazer lamentos torturados. Temerosas histórias locais avisam: aqueles que se aproximam demais são transformados em estátuas de gelo, formando um jardim macabro diante do palácio."

Fonte: Atlas de Arton, página 271.

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by WillianLimaSilveira** [comment]
**Nome do local:** Aldeia dos Centauros (ou Planície dos Centauros, de acordo com o Atlas de Arton)
**Reino:** Império de Tauron (Petrynia)
**Descrição sugerida:** A região descampada ao sul dos Bosques de Allihanna é território de um grande rebanho de centauros. Sua antiga líder, Odara de Allihanna, teve atuação importante durante a saga dos Rubis da Virtude. Após um incidente envolvendo a Tormenta, seu paradeiro é desconhecido; hoje o bando é liderado por sua aprendiz Hipólita. Diferente da antecessora, esta druida atrevida não se acanha em galopar até Nova Malpetrim e convocar aventureiros para matar monstros que eventualmente ameaçam a tribo.
**Fonte:** Atlas de Arton, p. 305.

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by WillianLimaSilveira** [comment]
**Nome do local:** Planalto dos Kobolds
**Reino:** Império de Tauron (Petrynia)
**Descrição sugerida:** em um planalto oculto nos Bosques de Allihanna, onde cavernas e túneis que atravessam o planalto escondem centenas, talvez milhares de criaturas. Como outros de seu tipo, estes também ficaram mais perigosos após o retorno de Kallyadranoch ao Panteão, mas ganharam ainda mais poder quando o dragão Xiuhdrake assumiu a liderança do bando. Sob comando do “enviado”, os kobolds atacam aldeias e fazendas vizinhas, capturando vítimas — especialmente crianças — para sacrifícios rituais ao Deus dos Dragões. 
**Fonte:** Atlas de Arton, p. 305.

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by WillianLimaSilveira** [comment]
**Nome do local:** Mansão de Zolkan
**Reino:** Império de Tauron (Petrynia)
**Descrição sugerida:** Situada à margem de um pântano próximo daVelha Malpetrim, pertenceu ao poderoso mago Zolkan, desaparecido há mais de 100 anos. Rumores dizem que a mansão é assombrada e maldita, algo atestado por heróis que a visitaram. Fantasmas e aparições infestam seus aposentos — que retornam magicamente ao normal logo após a passagem de aventureiros, não importando quanta coisa foi destruída ou quantas criaturas foram vencidas. Também dizem que Zolkan se tornou um monstro e devora todos que entram. Parte desse boato é verdade; o arcanista realmente se tornou “algo” graças a um experimento fracassado. Uma passagem mágica leva a um laboratório secreto longe dali, em uma caverna oculta, onde o monstro-Zolkan passa os dias tentando reverter seu atual estado.
**Fonte:** Atlas de Arton, p. 305.

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by WillianLimaSilveira** [comment]
**Nome do local:** Torre de Azazel
**Reino:** Império de Tauron (Petrynia)
**Descrição sugerida:** Situada em uma ilhota próxima à costa, esta grande torre pertence a Azazel, o Louco. Este velho de pele azulada — que muitos acreditam ser um tritão, embora não exista certeza sobre isso — mantém sua torre protegida por monstros marinhos, jamais tolerando a presença de invasores. Muitos navios foram afundados por suas criaturas porque se aproximaram demais. As histórias sobre a origem de Azazel seriam muitas; a mais conhecida diz que ele teria se apaixonado por uma sereia, mas foi rejeitado, e agora passa a vida pensando em formas de capturá-la e conquistar seu amor.
**Fonte:** Atlas de Arton, p. 305.

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by WillianLimaSilveira** [comment]
**Nome do local:** Fauchard
**Reino:** Império de Tauron (Petrynia)
**Descrição sugerida:** Distante quase mil quilômetros de Altrim, Fauchard é a segunda maior cidade de Petrynia. Por sua proximidade com a fronteira de Fortuna, é um importante porto comercial: recebe grande parte dos navios e cargas que chegam do Reinado.
**Fonte:** O Reinado d20, p. 86.

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by WillianLimaSilveira** [comment]
**Nome do local:** Kamalla
**Reino:** Império de Tauron (Petrynia)
**Descrição sugerida:** Este é o típico vilarejo de pescadores que encontramos aos milhares por Petrynia. O modo de vida é simples: os homens pescam no mar ou na costa, enquanto as mulheres cuidam das colheitas.
**Fonte:** O Reinado d20, p. 87.

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by ajjushi** [comment]
Nome do local: A Floresta Svalas
Reino (se tiver): Svalas
Descrição sugerida: Praticamente metade do território de Svalas é coberto por uma floresta densa e escura. Diziam que a floresta era assombrada pelos espíritos dos soldados svalanos que morreram durante a invasão.
Fonte: Atlas de Arton p. 233 no livro (p. 235 no pdf).

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by wessventura** [comment]
Nome do local: Vila Questor 
Reino (se tiver): Ruínas de Tyrondir
Descrição sugerida: Questor uma das raras comunidades costeiras. Situada em um dos raríssimos pontos navegáveis em toda a costa do reino, tem uma bem sucedida economia pesqueira. O artesanato local, à base de conchas, também é bastante apreciado... - proporcionando um crescimento rápido para a cidade.
Fonte: O Reinado - D20 p. 69

Foi a ultima meção feita.

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by wessventura** [comment]
Nome do local: Dagba
Reino (se tiver): Ruínas de Tyrondir
Descrição sugerida: Situada nos limites da Cordilheira de Kanter, nunca chegou a atingir grandes proporções. Atualmente a cidade está quase abandonada...
Fonte: O Reinado - D20 pg 68 e 69

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by GaryWellher** [comment]
Nome do local: Covil da Serpente
Reino (se tiver): Galrasia.
Descrição sugerida: O Covil da Serpente é um vulcão próximo à costa leste da ilha. Está extinto, mas não parece assim observado à distância, pois emana fumaça e chamas constantes. A cratera e seus túneis são agora habitados pela antiga princesa eiradaan Obassa, hoje transformada em uma criatura serpentiforme flamejante. O covil também é habitado por uma população de outros elementais do fogo.
Fonte: Atlas de Arton Pg. 352

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by GaryWellher** [comment]
Nome do local: Torre da Morte.
Reino (se tiver): Galrasia
Descrição sugerida: A Torre da Morte configura a maior estrutura erguida na ilha. No passado era muito bela, a habitação pessoal de Galron, de onde vigiava todo o Reino de Lena; hoje ergue-se enegrecida e macabra, vários prolongamentos parecidos com pernas insetoides nascendo em sua base — como se a torre inteira fosse algum inseto gigantesco, semienterrado. Seus muitos pavimentos abrigam bibliotecas, laboratórios e relicários contendo tesouros fabulosos, dizem. No entanto, também se especula que o próprio Galron ainda estaria ali, poderoso e rancoroso como nunca. Explorar a torre pode ser o desafio mais perigoso em toda a ilha.
Fonte: Atlas de Arton Pg. 352

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by GaryWellher** [comment]
Nome do local: Covil da Serpente
Reino (se tiver): Galrasia.
Descrição sugerida: A Floresta de Seda é morada de uma população de aranhas gigantes. Em suas árvores pendem véus abundantes de tecido sedoso, leve e forte, muito cobiçado pelos povos-trovão para manufatura de cordas. No entanto, suas partes mais profundas escondem cativeiros macabros onde vítimas — sobretudo coletores thera — são mantidas aprisionadas na teia, em longa agonia, provendo sangue e fluidos para as aranhas.
Fonte: Atlas de Arton Pg.352

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda (estava repetido com um acima)

**Comment by Ceabta** [comment]
Nome do local: Sordh
Reino (se tiver): Lomatubar
Descrição sugerida: Sordh é uma das centenas de cidades fantasma encontradas por todo o reino. Seus habitantes foram os primeiros humanos a mostrar sinais da doença. Antes que se pudesse descobrir a verdadeira extensão do perigo, a cidade inteira estava contaminada. Quase ninguém sobreviveu exceto alguns raros escolhidos, que acabaram se tornando Feiticeiros Vermelhos. Sordh era uma grande cidade, à margem do Rio Kodai. Tinha uma próspera relação comercial com o reino vizinho de Tollon, trocando mercadorias através de uma grande ponte. Quando a Praga veio, os tollianos apavorados destruíram a ponte a golpes de machado, cortando qualquer ligação com a cidade amaldiçoada. Hoje em dia Sordh é habitada apenas por mortos-vivos contaminados, que se arrastam pelas ruas e atacam qualquer coisa viva.
Fontes: O Reinado 3D&T - Parte 2, página 50

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by Ceabta** [comment]
Nome do local: Torre de Covariel
Reino (se tiver): Lomatubar
Descrição sugerida: Neste local o mago Thursten Covariel criou a Praga Coral, uma doença mágica e mortífera, para destruir as tribos
orcs que ameaçavam a região. O plano funcionara: os orcs foram dizimados, contudo, após essas décadas de paz, a praga voltou e contaminou todo tipo de humanoide — incluindo humanos e animais. Este costuma ser o primeiro local procurado por heróis que buscam uma cura. A torre já foi saqueada por muitos aventureiros, mas algumas passagens e câmaras secretas ainda não foram reveladas. Elas escondem laboratórios repletos de livros, anotações, ingredientes mágicos, amostras diversas da Praga e até mesmo algumas criaturas em cativeiro.
Fontes:  O Reinado 3D&T - Parte 2, página 52 + Atlas de Arton, página 297

**OBS:** Retirei e modifiquei algumas parte que falavam sobre a Praga Coral ser uma doença extraplar, que estão no "O Reinado 2", pois no "Atlas de Aton", que é mais recente, não cita sobre essa antiga origem, mas sim só como uma doença mágica.

**Reply by YuriAlessandro** [reply]
Adicionado! Obrigado pela ajuda.

**Comment by Ceabta** [comment]
Nome do local: Barud (Capital)
Reino (se tiver): Lomatubar
Descrição sugerida: Barud é a capital de Lomatubar e o único lugar do reino que ainda mantém alguma ordem após a devastação causada pela Praga Coral. Cercada por muralhas e protegida por uma guarda que examina rigorosamente todos os viajantes para impedir a entrada de infectados, a cidade conseguiu se manter livre da doença, mas vive em grande pobreza e escassez devido à destruição do interior do reino. Conquistada facilmente pelos minotauros nas Guerras Táuricas, Barud é governada pelo governador táurico Arlonus Meraxus, que governa como um tirano e pouco se importa com o restante da província, limitando-se a proteger a cidade enquanto o resto de Lomatubar permanece abandonado, tomado por doença, monstros e saqueadores.
Fontes: Atlas de Arton, página 297

**Comment by Ceabta** [comment]
Nome do local: Ralandar
Reino (se tiver): Lomatubar
Descrição sugerida: Em Ralandar encontramos uma típica cidade menor de Lomatubar, que resiste à Praga Coral como pode. Seus heróis locais protegem o lugar contra mortos-vivos e feras contaminadas, enquanto o povo tenta levar uma vida normal à base da agricultura e pecuária. Ralandar tem um templo devotado a Lena, a Deusa da Cura, e pelo menos duas clérigas residentes. Dizem que uma vítima da Praga levada para este templo ainda no primeiro estágio da doença pode ser salva com magias de cura menores o que normalmente não acontece fora do templo. Os camponeses de Lomatubar também desenvolveram o hábito de cultivar e consumir uma grande variedade de ervas medicinais, que supostamente teriam o poder de afastar a Praga. Cada cidade tem sua própria coleção de receitas. Embora essa "medicina natural" seja efetiva contra doenças comuns, nada disso funciona contra a Praga Coral.
Fontes: O Reinado 3D&T - Parte 2, página 50

**Comment by Ceabta** [comment]
Nome do local: Kurikondir
Reino (se tiver): Lomatubar
Descrição sugerida: A cidadela orc de Kurikondir é atualmente a maior comunidade desta raça existente no reino. Encravada em algum ponto das Montanhas Gull-Kor, ela é quase inteiramente subterrânea. Mesmo assim, seus habitantes foram dizimados pela Praga - e apenas em tempos recentes ela voltou a ser habitada. O rápido ciclo reprodutivo dos orcs permitiu repovoar Kurikondir em relativamente pouco tempo. Hoje eles são quase cinco mil, e aumentando. Sob o comando do rei orc Rugor Stonewill, a população da raça está crescendo em ritmo assustador. No momento, sua preocupação é retomar o domínio de suas próprias montanhas. Mais tarde, quando suas tropas forem mais numerosas, ele pretende avançar sobre os malditos humanos e dar ao povo orc sua grande vingança.
Fontes: O Reinado 3D&T - Parte 2, página 50

**Comment by wessventura** [comment]
Nome do local:Trodarr
Reino (se tiver): Florestas de Tollon
Descrição sugerida: A única cidade que sobreviveu quase intacta à passagem do Império de Tauron e sua retirada foi Trodarr, onde os anões de Tollon se estabeleceram. É talvez a única cidade anã de grande importância longe de Doherimm, para onde ex-aventureiros anões cansados do continente se mudavam quando não queriam voltar à Montanha de Ferro...a cidade dos anões lenhadores continua até hoje fornecendo madeira Tollon ao que sobrou do Império, a mercadores intrépidos e a quem mais ousar trazer uma caravana.
Fonte: Atlas de Arton, Página 280

**Comment by wessventura** [comment]
Nome do local: Arantur
Reino (se tiver): Florestas de Tollon
Descrição sugerida: O viajante ocasional pode perceber que, apesar de a travessia da floresta ser perigosa, ainda restam algumas comunidades de sobreviventes tollonienses em meio à névoa dos caminhos. A maior e mais emblemática é Arantur, formada por fugitivos da antiga capital Vallahim. Ao longo dos anos, membros do Conselho de Tollon mais inteligentes se uniram aos antigos Patrulheiros de Jeantalis, caçadores guardiões das rotas de comércio. Conhecedores do terreno, os patrulheiros ajudaram a retirar centenas de pessoas da capital e fundaram uma aldeia fortificada escondida em um mirante a leste da antiga capital.
Fonte: Atlas de Arton, Página 281

**Comment by wessventura** [comment]
Nome do local: Jardim Novo
Reino (se tiver): Florestas de Tollon
Descrição sugerida: Outra comunidade que vale ser mencionada é Jardim Novo, formada por fugitivos do ataque a Follen que seguiram ao sul e invadiram um estranho jardim de estátuas. Seu dono, um eremita mudo chamado Adhurraraim, os acolheu. Dizem que, desde então o jardim ganhou dezenas de novas estátuas, todas de legionários do Império de Tauron.
Fonte: Atlas de Arton, Página 281

**Comment by wessventura** [comment]
Nome do local: Fortaleza Grazomir
Reino (se tiver): Conflgração do Aço
Descrição sugerida: Por fora, este lugar parece um típico quartel da Supremacia, formado por um prédio baixo e largo, de paredes espessas e escuras, ao redor do qual espalham-se campos de treinamento, dormitórios e oficinas, tudo cercado por muros com torres de vigilância. Por dentro, porém, a Fortaleza Grazomir esconde horrores que a destacam de outros complexos militares da forma mais sórdida possível.
Fonte: Atlas de Arton, Página 188

**Comment by wessventura** [comment]
Nome do local: As Liças
Reino (se tiver): Conflgração do Aço
Descrição sugerida: Nas profundezas da Conflagração do Aço, existe uma clareira onde campeões se enfrentam em combate corporal dia e noite. Há cavaleiros de Bielefeld e soldados puristas, mas também guerreiros com estilos exóticos ou de regiões longínquas. Os combatentes chegam, desafiam alguém que julguem digno de suas capacidades e, se o desafio for aceito, começam a lutar.
Fonte: Atlas de Arton, Página 189

**Comment by wessventura** [comment]
Nome do local: Castelo de Pendor
Reino (se tiver): Conflgração do Aço
Descrição sugerida: Assim chamado por estar no topo de uma colina suave, este forte de pedra cinzenta era o centro de um feudo típico do sul de Bielefeld. Formado por terras aradas, pastos e uma aldeia cuja maior construção era um moinho de vento, era um lugar singelo, mas
próspero, governado por um casal de nobres justos
Fonte: Atlas de Arton, Página 188

**Comment by wessventura** [comment]
Nome do local: Campo de Experimentos
Reino (se tiver): Conflgração do Aço
Descrição sugerida: O Campo de Experimentos é um lugar onde conjuradores sem escrúpulos estudam e praticam todo tipo de magia proibida, como criação de monstros, dominação mental e invocação de demônios. Os arcanistas não possuem nenhuma aliança ou organização formal, mas seguem dois mandamentos: jamais se envolver nos experimentos dos outros e se unir em defesa do local caso ele seja atacado.
Fonte: Atlas de Arton, Página 190

**Comment by wessventura** [comment]
Nome do local: Campina Dourada
Reino (se tiver): Conflgração do Aço
Descrição sugerida: Esta aldeia é formada por algumas dúzias de casinhas de madeira, cercadas por pomares e pastos. Há uma estalagem, uma forja e uma pequena igreja. É uma comunidade bastante típica e, se estivesse em qualquer outro lugar de Arton, sequer apareceria nos
mapas. Porém, está na Conflagração do Aço — e aqui, um lugar vivo e habitado não é nada típico.
Fonte: Atlas de Arton, Página 191
